### ISSUE TITLE
Date Range with ranges

### ISSUE THREAD

**Issue by stevezau** [outbrain/pure-angular-date-range-picker#5]
Hi,

Question, I have set a bunch of pre-defined ranges (see below) which shows when i click on the date range picker. I then have to click custom to see the calendar. 

Is it possible to remove custom and just always show the calendar?

```
            ranges: [
                {name: 'Today', start: moment(), end: moment()},
                {name: 'Yesterday', start: moment().subtract(1, 'days'), end: moment().subtract(1, 'days')},
                {name: 'Last 7 Days', start: moment().subtract(6, 'days'), end: moment()},
                {name: 'Last 30 Days', start: moment().subtract(29, 'days'), end: moment()},
                {name: 'This Month', start: moment().startOf('month'), end: moment().endOf('month')},
                {name: 'Last Month', start: moment().subtract(1, 'month').startOf('month'), end: moment().subtract(1, 'month').endOf('month')},
            ]
```

**Comment by vlio20** [comment]
@stevezau, nope there is no option currently to do so. You are welcome to make a pull request or wait for the next release.

**Comment by vlio20** [comment]
resolved in 0.5.0 version
